src/Atmos/Model/moisture.jl

```diff
@@ -1,4 +1,5 @@
-export DryModel, EquilMoist
+import ..MoistThermodynamics: internal_energy
+export DryModel, EquilMoist, thermo_state
```

Comment:
`thermo_state` might potentially be used by Ocean later on?

Reply:
I believe this is the plan